Zotero/Controllers/ZoteroURIConverter.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  ZoteroURIConverter.swift
+//  Zotero
+//
+//  Created by Michal Rentka on 04.09.2024.
+//  Copyright © 2024 Corporation for Digital Scholarship. All rights reserved.
+//
+
+import Foundation
+
+final class ZoteroURIConverter {
```

Comment:
There seems to be some overlap of concerns with `CustomURLController`. At some later point we can consider merging them.

Reply:
Custom urls are similar (they have the same library/group part, but otherwise they are different, so I created different controller for it.

Examples:
URI: https://zotero.org/user/{user_id}/{key}
Custom url: zotero://select/library/items/{key}